### PR TITLE
(chore) Generate config reference for 2.8

### PR DESCRIPTION
**Pull request by lena-larionova**
### Summary
Generate configuration reference for Gateway 2.8 from `kong.conf.default`.

Manually document `admin_auto_create_rbac_token_disabled`, as it is missing from the source file in the kong-ee repo. This one will need tech review.

### Reason
Chore PR.

### Testing
https://deploy-preview-3717--kongdocs.netlify.app/gateway/2.8.x/reference/configuration/

### CODE REVIEW

**Comment by lena-larionova** [comment]
@mayocream can you please review the doc entry for `admin_auto_create_rbac_token_disabled` (based on https://github.com/Kong/kong-ee/pull/2988)? Normally it would be generated from [kong.conf.default](https://github.com/Kong/kong-ee/blob/master/kong.conf.default), but I noticed this parameter is missing from that file.

File: app/gateway/2.8.x/reference/configuration.md [hunk 1]
```diff
@@ -2302,6 +2314,19 @@ Example for `basic-auth`:
...
 ---
 
+#### admin_auto_create_rbac_token_disabled
+{:.badge .enterprise}
+
+If `admin_gui_auth` is set to `openid-connect`, enable or disable RBAC token
+creation when automatically creating admins with OpenID Connect.
+
+* Set to `true` to enable automatic token creation for admins
+* Set to `false` to disable automatic token creation for admins
```
Comment (mayocream) [inline app/gateway/2.8.x/reference/configuration.md:2324]:
```suggestion
* Set to `true` to disable automatic token creation for admins
* Set to `false` to enable automatic token creation for admins
```

**Comment by mayocream** [comment]
> @mayocream can you please review the doc entry for `admin_auto_create_rbac_token_disabled` (based on https://github.com/Kong/kong-ee/pull/2988)? Normally it would be generated from [kong.conf.default](https://github.com/Kong/kong-ee/blob/master/kong.conf.default), but I noticed this parameter is missing from that file.

I am sorry I didn't noticed that the option added is not in the kong.conf.default file.

The new added option is in admin_gui_auth_conf on this page: 
https://docs.konghq.com/gateway/2.7.x/configure/auth/kong-manager/oidc-mapping/

**Comment by lena-larionova** [comment]
> I am sorry I didn't noticed that the option added is not in the kong.conf.default file.
> The new added option is in admin_gui_auth_conf on this page:
https://docs.konghq.com/gateway/2.7.x/configure/auth/kong-manager/oidc-mapping/

Ack. I'll move the info to that page.

Is `admin_auto_create_rbac_token_disabled` then an OpenID connect property?

**Comment by lena-larionova** [comment]
@mayocream moved it, see https://github.com/Kong/docs.konghq.com/pull/3727.

**Review by GayleNeumann — APPROVED** [review]
LGTM

**Pull request merged**